api/resources_portal/test/dev_data/test_data_readme.txt

```diff
@@ -0,0 +1,45 @@
+The test data describes the following situation:
+
+There are three users:
+- PrimaryProf
+- PostDoc
+- SecondaryProf
+
+There are fours organizations which have these users as members:
+- PrimaryLab, owned by PrimaryProf
+- PrimaryProfPersonalOrg, owned by PrimaryProf with PostDoc as a member
+- PostDocOrg, owned by PostDoc
+- SecondaryProfOrg, owned by SecondaryProf
+
+There are two grants:
+- A grant for melanoma research, given to PrimaryProf
+- A grant for tumor growth research, given to SecondaryProf
+
+There are eight materials:
+	Four contributed by PrimaryProf:
+	- A Plasmid
+	- A Protocol
+	- A Model Organism (zebrafish)
+	- A Model Organism (mouse)
+
+	Four contributed by SecondaryProf:
+	- A Stone Tablet (using the "other" template)
+	- A Cell Line
+	- A Dataset
+	- A PDX
+
```

Comment:
Are all of these materials listed? or are some of them imported?

Reply:
I don't think the difference between listed and imported materials is indicated anywhere in the database currently. Should it be? As I understand it, importing and listing are just two ways of filling out the same data fields. Maybe the difference is that imported materials have a URL, and listed ones don't?

Reply:
Yes, the imported fields do have an URL and Listed ones don't. 
Import forms have a different set of fields than the listed fields. Usually, it is only a subset of the fields but some like GEO/SRA depending on whether it is under embargo or not, the fields are different.

Reply:
Would it be helpful in your testing if I found real URLs with examples of the relevant materials? The current URLs are just dummy data.

Reply:
> Import forms have a different set of fields than the listed fields. Usually, it is only a subset of the fields but some like GEO/SRA depending on whether it is under embargo or not, the fields are different.

@dvenprasad can you elaborate on that more? Do you have examples we could look at? (Or alternatively, teach me to fish for that. Should I be able to find that for myself?)

Reply:
Okay, so I went looked through the resources. All of the materials have the fields from their corresponding List forms, which is good.

- I would suggest leaving the URL fields blank and this set of materials will reflect that they were all listed. 

- I also suggest adding another set of materials representing imported materials, where the URL field is populated and have fields from the corresponding Import forms (see list for imported materials https://github.com/AlexsLemonade/resources-portal/issues/3#issuecomment-605287481)

Let me know if need help finding the import forms.

Reply:
@kurtwheeler Example: this is the form for Listing a Plasmid: https://invis.io/NAVYSEBXJCF#/404857864_list-Resource-Form-Plasmid

This is importing a Plasmid i.e AddGene
https://invis.io/NAVYSEBXJCF#/404857852_import-Resource-Form-Add-Gene

So if you start here https://invis.io/NAVYSEBXJCF#/406884962_import-Resource-Geo-Sra-Extracted-Pubmed-Not-Editable and keep skipping to the next screen, you should be all of the import forms.

Reply:
Thanks Deepa!